data/json/techniques.json

```diff
@@ -267,7 +267,7 @@
         "name" : "Cross Counter",
         "min_unarmed" : 5,
         "unarmed_allowed" : true,
-        "crit_tec" : true,
+        "dodge_counter": true,
```

Comment:
Pretty sure this isn't supposed to be a dodge counter. It requires an on-dodge buff.